client/apollo/react/src/Form/checkbox/checkboxCard/types.d.ts

```diff
@@ -0,0 +1,11 @@
+import type { ComponentProps, ComponentType } from "react";
+import type { Icon } from "../../../indexLF";
```

Comment:
tu peux importer `IconProps` directement de _IconCommon.tsx_ et l'utiliser pour typer `IconComponent`
```suggestion
import type { IconProps } from "../../../Icon/IconCommon";
```